### ISSUE TITLE
"Three steps" explanation is not a list

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4025]
## Area or Component
Sign up form

## Quick Summary
The instructions are a list, but are not encoded as a list (i.e. with `<li>`)

## Accessibility Violation
Visual list is not marked up as list: https://cdc-devtools.dequecloud.com/rule-help/en/semantic-list

## Action Requested
Change the structure of the instructions to be a list.

## Screenshots (optional)
1, 2, 3 here are not a list:
![Screen Shot 2022-07-21 at 1.44.46 PM.png](https://images.zenhubusercontent.com/604699209bfd9306195da0d7/7418734d-51c2-4cbe-9f19-3e3d210c1c72)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=33:33
